ldms/src/core/ldms.c

```diff
@@ -6145,9 +6145,10 @@ const char *ldms_msg_event_type_sym(enum ldms_msg_event_type t)
 		[LDMS_MSG_EVENT_SUBSCRIBE_STATUS] =
 			"LDMS_MSG_EVENT_SUBSCRIBE_STATUS",
 		[LDMS_MSG_EVENT_UNSUBSCRIBE_STATUS] =
-			"LDMS_MSG_EVENT_UNSUBSCRIBE_STATUS",
+			"LDMS_MSG_EVENT_UNSUBSCRIBE_STATUS"
 	};
-	if (t < LDMS_MSG_EVENT_LAST)
+	if (t <= LDMS_MSG_EVENT_CLIENT_CLOSE) {
```

Comment:
@nick-enoent please change this to:
```C
if (t < sizeof(tbl) / sizeof(tbl[0])) {
```
If we do it that way, we'll never need to change this code if we add another event.